core/src/avm2/globals/flash/filesystem/File.as

```diff
@@ -0,0 +1,46 @@
+package flash.filesystem
+{
+    import flash.net.FileReference;
+
+    public class File extends FileReference
+    {
+        private native function init(path: String = null);
```

Comment:
`path: String = null`- a default parameter is unnecessary since the constructor will always call it with an argument